api/v1alpha1/account_types.go

```diff
@@ -82,6 +82,35 @@ const (
 	Elasticloadbalancing SupportedServiceQuotaServices = "elasticloadbalancing"
 )
 
+type OptInRegions map[string]*OptInRegionStatus
+
+type OptInRegionStatus struct {
+	RegionCode string             `json:"regionCode"`
+	Status     OptInRequestStatus `json:"status"`
+}
+type OptInRequestStatus string
+
+const (
+	OptInRequestTodo     OptInRequestStatus = "TODO"
+	OptInRequestEnabling OptInRequestStatus = "ENABLING"
+	OptInRequestEnabled  OptInRequestStatus = "ENABLED"
+)
+
+type SupportedOptInRegions string
+
+const (
+	CapeTownRegion  SupportedOptInRegions = "af-south-1"
+	MelbourneRegion SupportedOptInRegions = "ap-southeast-4"
+	HyderabadRegion SupportedOptInRegions = "ap-south-2"
+	MilanRegion     SupportedOptInRegions = "eu-south-1"
+	ZurichRegion    SupportedOptInRegions = "eu-central-2"
+	HongKongRegion  SupportedOptInRegions = "ap-east-1"
+	UAERegion       SupportedOptInRegions = "me-central-1"
+	SpainRegion     SupportedOptInRegions = "eu-south-2"
+	BahrainRegion   SupportedOptInRegions = "me-south-1"
+	JakartaRegion   SupportedOptInRegions = "ap-southeast-3"
+)
```

Comment:
I'd advise against having this in the code, especially in the API for a few reasons:

1. Every change to add a new AWS region will now require a code change and promotion
2. Being in the API package, which other services (hive, ocm, etc) consume - we could run into issues with downstream consumers not updating, and if we ever make changes for any reason we'll have to run through deprecation procedures, etc.

Mostly - it's a large maintenance cost to add these to the code, and we'll _also_ need subsequent SOP updates to track these changes for our future selves.